### PR TITLE
Handle case where no Authorization header is provided

### PR DESCRIPTION
**Pull request by acrellin**

### CODE REVIEW

File: app/access.py [hunk 1]
```diff
@@ -34,6 +34,8 @@ def wrapper(self, *args, **kwargs):
             if self.current_user is not None:
                 if not self.current_user.is_active():
                     raise tornado.web.HTTPError(403, "User account expired")
+            else:
+                raise tornado.web.HTTPError(403, "Credentials missing or malformed")
```
Comment (stefanv) [inline app/access.py:38]: This should be a 401. Can we provide more explicit instructions to the user?  Something like: `Credentials malformed; expected header of form "authorization: token abc123"`

Comment (stefanv) [reply app/access.py:38]: We could probably catch this more explicitly on line 24 (`token_header.replace`)?

**Comment by acrellin** [comment]
@stefanv updated (though not sure what you're suggesting with the latter comment)

**Comment by stefanv** [comment]
I just meant that we are doing a generic catch, instead of checking the form of the header explicitly.

**Comment by acrellin** [comment]
Ah, don't lines 22-23 do just that?

**Comment by stefanv** [comment]
Hrm, yes, you're right; so what is this error that we are catching?  How will it typically arise?

**Comment by stefanv** [comment]
Thanks!

**Pull request merged**